pluginserver.go

```diff
@@ -174,14 +174,14 @@ func getSchemaDict(t reflect.Type) schemaDict {
 	case reflect.Float32, reflect.Float64:
 		return schemaDict{"type": "number"}
 
-	case reflect.Array:
+	case reflect.Slice:
 		elemType := getSchemaDict(t.Elem())
 		if elemType == nil {
 			break
 		}
 		return schemaDict{
 			"type":     "array",
-			"elements": schemaDict{"type": elemType},
+			"elements": elemType,
```

Comment:
This change doesn't look correct at a first glance. Could you elaborate here @longquan0104? Perhaps elaborate on the error it fixes, with a stacktrace?